sle_prjmgr_tools/cli.py

```diff
@@ -126,7 +129,15 @@ def main():
     args = PARSER.parse_args()
     if "func" in vars(args):
         # Run a subprogramm only if the parser detected it correctly.
-        args.func(args)
+        try:
+            args.func(args)
+        except urllib.error.URLError as url_error:
+            if "name or service not known" in str(url_error).lower():
+                print(
+                    "No connection to one of the tools. Please make sure the tools are available before executing the"
```

Comment:
Should be "...Please make sure **_connection is_** available..."?

Reply:
I will adjust it. Thanks for the hint.